src/karts/abstract_kart.hpp

```diff
@@ -459,6 +476,9 @@ class AbstractKart : public Moveable
      *  defined even if the kart is flying. */
     virtual const Vec3& getNormal() const = 0;
     // ------------------------------------------------------------------------
+    /** Returns the position 0,25s before */
```

Comment:
0.25, not ","

Reply:
Right. In french, we use "," rather than "." as a separator there (and I can tell you that calc is very unhappy if I try to use the english way). As it is in a comment, it went fine through the compilator. Feel free to change it. I won't fix it unless I need to change other things in that file though, such a minor typo don't deserve that effort.